doc/user/content/manage/monitor/grafana.md

```diff
@@ -0,0 +1,188 @@
+                SELECT
+                   name::text AS replica_name,
+                   cluster_id::text AS cluster_id,
+                   memory_percent::float AS memory_percent
+                FROM mz_cluster_replicas r
+                JOIN mz_internal.mz_cluster_replica_utilization u ON r.id=u.replica_id;
+   ```
+
+4. Once you are done with the Prometheus SQL Exporter configuration,
+   follow the intructions in the [`sql_exporter` repository](https://github.com/justwatchcom/sql_exporter#getting-started)
+   to run the service using the configuration file from the previous step.
+
+## Step 2. Set up a scraper
+
+To scrape the metrics available in the Prometheus SQL Exporter endpoint, you
+must then set up a [Grafana Agent](https://grafana.com/docs/agent/latest/?pg=oss-agent) for Grafana cloud, or [Prometheus](https://prometheus.io/download/) for the self-managed version:
+
+{{< tabs >}}
+{{< tab "Cloud">}}
+
+1. Follow the [instructions to install and run a Grafana Agent](https://grafana.com/docs/agent/latest/static/set-up/install/)
+   in your host.
+
+2. To configure a [Prometheus scrape](https://grafana.com/docs/grafana-cloud/monitor-infrastructure/metrics/metrics-prometheus/)
+   for the Grafana Agent installed in the previous step, create and edit the [agent configuration file.](https://grafana.com/docs/agent/latest/static/configuration/create-config-file/)
+
+   **Filename:** agent.yaml
+   ```yaml
+      ...
+      remote_write:
+         - url: <REMOTE_WRITE_URL>
+         basic_auth:
+            username: <USERNAME>
+            password: <PASSWORD>
+   ```
+
+   **Tip:** see [this sample](https://github.com/MaterializeInc/demos/blob/main/integrations/grafana/cloud/agent.yaml)
+   for all available configuration options.
+
+   For more details on how to configure, run and troubleshoot Grafana Agents, see the [Grafana documentation](https://grafana.com/docs/agent/latest/).
+
+   <br/>
+   <details><summary>Video for generating configuration values for the first time.</summary>
+
+   ![Gif](https://github.com/MaterializeInc/demos/assets/11491779/e512a95f-e3c6-433d-bc8f-6f5138b08115)
+
+   </details>
+
+{{< /tab >}}
+{{< tab "Self-managed">}}
+1. Follow the [instructions to install and run Prometheus](https://prometheus.io/docs/prometheus/latest/installation/)
+   in your host.
+
+2. To configure a [Prometheus scrape](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config) edit the
+   `prometheus.yml` file as follows:
+
+   **Filename**: prometheus.yml
+   ```yaml
+      ...
+      - job_name: sql_exporter
+         scrape_interval: 15s
+         static_configs:
+            - targets: ['sql-exporter:9237']
+            labels:
+               instance: sql_exporter
+   ```
+
+     **Tip:** see [this sample](https://github.com/MaterializeInc/demos/blob/main/integrations/grafana/local/prometheus.yml) for all available configuration options.
+
+3. Add **Prometheus** as a new source to Grafana.
```

Comment:
```suggestion
3. Follow [the instructions](https://grafana.com/docs/grafana/latest/datasources/prometheus/) to add **Prometheus** as a new data source in Grafana.
```

Reply:
We have a sample data source file [here](https://github.com/MaterializeInc/demos/blob/main/integrations/grafana/local/misc/datasources/prometheus.yml) that we should mention?

Reply:
Yes!